scripts/apps/vocabularies/views/vocabulary-config-modal.html

```diff
@@ -52,6 +52,15 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
                          ng-show="vocabularyForm.$error.maxlength" translate>The helper text must not exceed 120 characters.</div>
                 </div>
 
+                <div class="sd-line-input" ng-if="matchFieldTypeToTab('vocabularies', vocabulary.field_type)">
+                    <label for="cv_selection_type" class="sd-line-input__label" translate>Show to users as:</label>
+                    <select class="input-medium" ng-model="vocabulary.selection_type">
+                        <option value="single selection" translate>single selection</option>
```

Comment:
Create a constant with selection types. Iterate it in this select and use it for comparison in other places.

Reply:
Done